cob_default_robot_config/cob4-2/head_joint_configurations.yaml

```diff
@@ -5,16 +5,16 @@ default_vel: 0.4
 
 home: [[0.0, 0.0, 0.0]]
 
-front: [[0.0, 0.0, 0.0]]
-front_down: [[1.57, -3.14, 1.57]]
-front_down_twisted: [[1.57, -3.14, -1.57]]
+front: [[0.0, 0.0, 3.14]]
```

Comment:
why are those poses different to simulation? @ipa-fxm updated the poses for cob4-1 in sim some weeks ago